### PR TITLE
not completely done with tests yet, but adding some

### PR DESCRIPTION
**Pull request by loganballard**
more tests are here, but more importantly, this implements the "clear screen" functionality that makes the terminal much more usable.  Doesn't fix the test_suite ticket, but still useful to have this in `master` while we work

### CODE REVIEW

File: src/server_input_handler.rs [hunk 1]
```diff
@@ -150,6 +150,50 @@ fn start_new_game(
...
+//    );
+//    assert!(id_game_map_m.lock().unwrap().contains_key(&client_id));
+//    assert_eq!(
+//        *(id_game_map_m.lock().unwrap().get(&player_one_id).unwrap()),
+//        game_id
+//    );
+//    assert_eq!(
+//        *(id_game_map_m.lock().unwrap().get(&client_id).unwrap()),
+//        game_id
+//    );
```
Comment (rooneyshuman) [inline src/server_input_handler.rs]: What's going on here?

Comment (loganballard) [reply src/server_input_handler.rs]: lol

Comment (loganballard) [reply src/server_input_handler.rs]: bad QA

Comment (loganballard) [reply src/server_input_handler.rs]: fixed

**Pull request merged**